### PR TITLE
Add Firebase function call to update RSVPs

### PR DESCRIPTION
**Pull request by taran-gill**

### CODE REVIEW

File: my/src/types.ts [hunk 1]
```diff
@@ -116,9 +116,7 @@ export interface StatusModel {
     location: string;
     email: string | null;
   };
-  criticalError: boolean;
-  hasResponded: boolean;
-  confirmation: boolean;
```
Comment (taran-gill) [inline my/src/types.ts:121]: These three booleans are dead code

File: my/src/views/Status.vue [hunk 2]
```diff
@@ -288,29 +288,9 @@
...
-          top
-          color="error"
-          right
-          :timeout="10000"
-        >
-          Could not establish connection to the server. Consider refreshing or
-          attempting to access the page on mobile. If the issue persists contact
-          us at hello@deltahacks.com.
-          <v-btn color="white" flat @click="criticalError = false">Close</v-btn>
+        <v-snackbar top right :color="snack.color" v-model="snack.visible" :timeout="snack.timeout">
```
Comment (amirafzali) [inline my/src/views/Status.vue:291]: nice cleanup 😎

File: my/src/types.ts (hunk 1 above)
Comment (amirafzali) [reply my/src/types.ts:121]: F in the chat

File: my/src/views/Status.vue [hunk 3]
```diff
@@ -457,7 +441,7 @@ export default Vue.extend({
     // Grabs the application from where its store in firebase
     fetchFromFirebase(): Promise<any> {
       return this.$store.state.db
-        .collection('DH6')
+        .collection('DH7')
```
Comment (amirafzali) [inline my/src/views/Status.vue:444]: !

**Review by amirafzali — APPROVED** [review]
LGTM 👍

**Review by dhruv423 — APPROVED** [review]

**Pull request merged**